src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -346,9 +346,11 @@ public function getReflectionClass($class, $throw = true)
         try {
             if (isset($this->classReflectors[$class])) {
                 $classReflector = $this->classReflectors[$class];
-            } else {
+            } elseif ($this->trackResources) {
                 $resource = new ClassExistenceResource($class, false);
                 $classReflector = $resource->isFresh(0) ? false : new \ReflectionClass($class);
+            } else {
+                $classReflector = new \ReflectionClass($class);
```

Comment:
Should be false if the clas/interface/traits doesn't exist

Reply:
happens in catch no? L359 here.